spec/api-browser-window-spec.js

```diff
@@ -1145,8 +1145,11 @@ describe('BrowserWindow module', function () {
         const initialWebContents = webContents.getAllWebContents()
         ipcRenderer.send('prevent-next-new-window', w.webContents.id)
         w.webContents.once('new-window', () => {
-          assert.deepEqual(webContents.getAllWebContents(), initialWebContents)
-          done()
+          // We need to give it some time so the windows get properly disposed (at least on OSX).
+          setTimeout(() => {
+            assert.deepEqual(webContents.getAllWebContents(), initialWebContents)
+            done()
+          }, 100)
```

Comment:
I'm not sure this should be changed, this spec wasn't failing on master CI, was it failing only on this branch?